index.html

```diff
@@ -4861,6 +4862,122 @@ <h3>Fitness for Purpose</h3>
       </section>
     </section>
 
+  </section>
+
+  <section class="appendix informative">
+    <h2>Base Context</h2>
+      The base context located at <code>https://www.w3.org/2018/credentials/v1</code> with SHA256(<code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code>) can be used to implement a local cached copy.  The aforementioned context is provided in his section for convience.
```

Comment:
```      The base context located at <code>https://www.w3.org/2018/credentials/v1</code> with SHA256(<code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code>) can be used to implement a local cached copy.  The aforementioned context is provided in his section for convience.
```
->
```
The base context, located at <code>https://www.w3.org/2018/credentials/v1</code> 
with SHA-256 digest 
<strong><code>63C0C0F374AB61F4B810BEF8A6DEE5F83C7D845992EB920CAF2505C4C99BB91C</code></strong>,
can be used to implement a local cached copy.  The aforementioned context is 
provided in this section for convenience.
```
* added line breaks
* tweaked styling around and naming of `SHA-256` (properly requires hyphen)
* `convience` and `his` spelling -> `convenience` and `this`